src/telliot_core/data/contract_directory.json

```diff
@@ -56,11 +56,12 @@
     "address": {
       "137": "0xFd45Ae72E81Adaaf01cC61c8bCe016b7060DD537",
       "122": "0x66c67C69bed74385A4fFbF3986B3e4736c54a5e1",
-      "80001": "0x41b66dd93b03e89D29114a7613A6f9f0d4F40178",
-      "3": "0x7B8AC044ebce66aCdF14197E8De38C1Cc802dB4A",
-      "69": "0xE13a7288a27a35d2481874620abC04a9C2473Fde",
+      "80001": "0x840c23e39F9D029fFa888F47069aA6864f0401D7",
+      "3": "0xbB97C038c338c3DcAF06D5be3B4A3e0B24835f9C",
```

Comment:
Is this one supposed to be `0x7B8AC044ebce66aCdF14197E8De38C1Cc802dB4A`?

Reply:
yes